### PR TITLE
`setupRenderingTest`, etc.

### PR DESCRIPTION
**Pull request by pittst3r**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "postpublish": "git push origin master --tags"
   },
   "dependencies": {
-    "@glimmer/compiler": "v0.24.0-beta.5",
+    "@glimmer/compiler": "^0.24.0-beta.6",
```
Comment (rwjblue) [inline package.json]: This should be a dev dep I believe

File: package.json [hunk 2]
```diff
@@ -16,9 +16,9 @@
...
     "@glimmer/component": "^0.4.0",
```
Comment (rwjblue) [inline package.json]: dev dep

File: package.json [hunk 3]
```diff
@@ -16,9 +16,9 @@
...
-    "@glimmer/wire-format": "v0.24.0-beta.5",
+    "@glimmer/wire-format": "^0.24.0-beta.6",
```
Comment (rwjblue) [inline package.json]: dev dep

**Review by rwjblue — COMMENTED** [review]
Can you also make @types/qunit a dep (not a dev dep)?

**Review by rwjblue — APPROVED** [review]

**Comment by Turbo87** [comment]
@rwjblue is this bound to QUnit or can we make it compatible with Mocha again too?

**Comment by rwjblue** [comment]
@Turbo87 - My plan is roughly the same setup we have in Ember.  This was just a first pass to get something functional.

**Comment by pittst3r** [comment]
Confirm. There is a bunch of work left to do. This is very MVP.

**Pull request merged**